src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/pipeline/JiraSendDeploymentInfoStep.java

```diff
@@ -102,6 +101,15 @@ public StepExecution start(final StepContext stepContext) throws Exception {
         return new JiraSendDeploymentInfoStepExecution(stepContext, this);
     }
 
+    public List<String> getServiceIds() {
+        return serviceIds;
+    }
+
+    @DataBoundSetter
+    public void setServiceIds(final List<String> serviceIds) {
```

Comment:
This is a new optional step parameter - it should be also added to `src/main/resources/com/atlassian/jira/cloud/jenkins/deploymentinfo/pipeline/JiraSendDeploymentInfoStep/config.jelly`, so it can seen in the Pipeline Snippet Generator.

The same goes for the new "state" optional param - I think I overlooked this in my previous review